### DISCUSSION TITLE
From Robots to Robot Studio workflow problems

### DISCUSSION THREAD

**Discussion by HelledJack** [visose/Robots#293] [Q&A] [answered]
Hi there,

I just started using the Robots plug-in in Grasshopper, but when I try to import the code into Robot Studio, it doesn't work. It seems like the points where the planes are defined disappear, but when I run the simulation in Rhino/Grasshopper, everything works fine.

I haven't been able to figure out what I'm missing or doing wrong.

I'm stuck; please help me!

[MILLING1_T_ROB1.txt](https://github.com/user-attachments/files/17467042/MILLING1_T_ROB1.txt)

**Comment by visose** [comment]
The points seem to be there in the text file you shared. Not sure exactly how you are importing it into RobotStudio, but uou don't really need to use it. You can use the Remote component to upload the program into the controller, or you can use the Save component and copy it using a USB stick.
I presume you used the Save program and then uploaded it to the controller using RobotStudio? Or did you copy paste the code and edited a .mod file?

**Comment by visose** [comment, marked as answer]
Closing this as stale after the follow-up question. The shared text file did contain the point data, so the likely issue was the RobotStudio import/copy-paste workflow rather than Robots omitting points. For current workflows, use Remote or Save Program and transfer the generated module files directly. Please reopen with exact RobotStudio import steps and unedited generated files if it still fails.
